src/compiler/moduleNameResolver.ts

```diff
@@ -5,7 +5,7 @@ namespace ts {
 
     /* @internal */
     export function trace(host: ModuleResolutionHost, message: DiagnosticMessage, ...args: any[]): void;
-    export function trace(host: ModuleResolutionHost, message: DiagnosticMessage): void {
+    export function trace(host: ModuleResolutionHost, _message: DiagnosticMessage): void {

```

Comment:
consider removing `_message` all together from the implementation signature,  `export function trace(host: ModuleResolutionHost): void { .. }`